met-web/src/components/engagement/form/EngagementWidgets/Documents/CreateFolderForm.tsx

```diff
@@ -22,11 +22,11 @@ const CreateFolderForm = () => {
     const [formError, setFormError] = useState(initialFormError);
     const widget = widgets.find((widget) => widget.widget_type_id === WidgetType.Document);
 
-    const validate = () => {
+    const valid = () => {
```

Comment:
It should be either isValid or validate() ..valid() doesnt give much info

Reply:
ill change it to isValid then